modules/log/commerce_log.module

```diff
@@ -20,3 +22,8 @@ function commerce_log_preprocess_commerce_order(&$variables) {
     '#title' => t('Order activity'),
   ];
 }
+
+function commerce_log_entity_type_alter(array &$entity_types) {
+  /** @var $entity_types \Drupal\Core\Entity\EntityTypeInterface[] */
+  $entity_types['commerce_payment']->setListBuilderClass(PaymentListBuilder::class);
```

Comment:
I don't care if the module is enabled, if it isn't this will just get ignored.

Reply:
If the module is disabled there won't be an $entity_types['commerce_payment'] object, causing a crash, no?

Reply:
Ah, that's true. I ignored that important item. I was still in d7 array mode.